carbon/config.go

```diff
@@ -18,6 +19,36 @@ const (
 	MetricEndpointLocal = "local"
 )
 
+type bytesSize struct {
```

Comment:
Sorry, just figured out, there's a dedicated package for such types, https://github.com/lomik/carbon-clickhouse/tree/master/helper/config